website/content/en/docs/building-operators/golang/testing.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Testing your Operator project
+linkTitle: Testing with EnvTest
+description: Learn how to ensure the quality of your Operator project 
+weight: 70
+---
+
+## Overview
+
+The Operator SDK project recommends using controller-runtime's [envtest][envtest] to write tests for your Operators projects because it has a more active contributor community, it has become more mature than Operator SDK's test framework, and it does not require an actual cluster to run tests, which can be a huge benefit in CI scenarios. 
+
+## Using EnvTest
+
+You will see that `controllers/suite_test.go` is created when a controller is scaffolded by the tool. This file contains boilerplate for executing integration tests using [envtest][envtest] with [ginkgo](https://onsi.github.io/ginkgo/) and [gomega](https://onsi.github.io/gomega/).
+
+It means that you will implement your controller's tests in go using this stack and it will be called by the `suite_test.go` via go tool such as: 
+
+```shell
+go test controllers/ -v -ginkgo.v
+```
+
+The projects generated by using the SDK tool have a Makefile which will contains the target `tests` executes the tests when you make tests. Note that when you run`make docker-build IMG=<some-registry>/<project-name>:<tag>` the test also will be executed. 
```

Comment:
maybe "which contains the target `tests` which executes when you run `make test`. Note that this target will also execute when you run `make docker-build IMG=<some-registry>/<project-name>:<tag>"